system/jlib/jlog.cpp

```diff
@@ -108,6 +108,7 @@ class LoggingFieldColumns
         { MSGFIELD_user,      "UserID  " },
         { MSGFIELD_component, "Compo " }
     };
+    const char * notQuotedPrefix = "// NOTQUOTED";
```

Comment:
I have done the change like this mostly so that it is backwardly compatible.  So that if they have already upgraded to 7.x, the fields will be quoting messages straight away and not have the fix working only after the logs have been rotated.

Also, MSG_quote and the other MSGFIELD_* are not really the same thing. MSGFIELD_quote is more like a flag to say how message columns are outputted and the others are real columns.

Reply:
ok, that's good, but the original questions remain...  What is "//" for and why is it necessary. Also, why don't we keep track of the quoted/unquoted column name in the table. @shamser